src/theme.js

```diff
@@ -1,4 +1,20 @@
-var jQuery = (typeof(window) != 'undefined') ? window.jQuery : require('jquery');
+/**
+ * Inject jquery if isn't present already.
+ *
+ * @param {function} fn - Function to be executed after jquery has been loaded.
+ */
+function injectJQuery(fn) {
```

Comment:
Webpack does this natively. We use this import method in other places for similar conditional imports.

https://webpack.js.org/api/module-methods/#import-1